notebooks/user_guide/Resampling_Grids.ipynb

```diff
@@ -0,0 +1,460 @@
+    "Since the grid has 2D coordinate arrays the regridded data will be wrapped in and displayed as a QuadMesh:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "target = gv.Dataset(grid, kdims=['lon', 'lat'])\n",
+    "weighted_regrid(images, target=target, streams=[])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Curvilinear Grids"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Curvilinear grids are another very common mesh type, which are usually defined by multi-dimensional coordinate arrays:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds = xr.tutorial.load_dataset('rasm')\n",
+    "# Adjust the longitudes so they are in the range -180 to 180\n",
+    "ds.xc.values[ds.xc.values>180] -= 360 \n",
+    "ds"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Just like the rectilinear grids GeoViews understands this kind of data natively. So we again wrap this dataset in a ``gv.Dataset`` and define a fixed range for the ``Tair`` values:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gvds = gv.Dataset(ds).redim.range(Tair=(-25, 25))\n",
+    "gvds"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can plot this data directly as a ``gv.QuadMesh``, however this is generally quite slow, especially when we are working with bokeh where each grid point is rendered as a distinct polygon. We will therefore downsample the data by a factor of 3 along both dimensions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%opts Image QuadMesh (cmap='RdBu_r')\n",
+    "quadmeshes = gvds.to(gv.QuadMesh, ['xc', 'yc'], dynamic=True)\n",
+    "quadmeshes.map(lambda x: x.clone(x.data.Tair[::3, ::3]), gv.QuadMesh) * gv.feature.coastline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The problem is less severe when plotting using matplotlib but even then plotting can be fairly slow given a large enough mesh."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%output backend='matplotlib' size=300\n",
+    "%%opts QuadMesh (cmap='RdBu_r')\n",
+    "quadmeshes * gv.feature.coastline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we want to explore a very large grid it therefore often makes sense to resample the data onto a rectilinear grid, which can be rendered much more efficiently. Once again we have the option of using the datashader based approach or the more correct xESMF based approach."
```

Comment:
I'd say "more accurate" rather than "more correct"; there are varying levels of accuracy but correctness is Boolean.  Neither approach is unequivocally "correct"; one can always use more and more complex models (given that the earth isn't actually precisely spherical, for instance).